packages/devtools_app/lib/src/shared/analytics/constants.dart

```diff
@@ -201,13 +197,8 @@ class MemoryEvent {
 /// Areas of memory screen, to prefix event names, when events are emitted
 /// by a widget used in different contexts.
 enum MemoryAreas {
-  snapshotSingle('single'),
-  snapshotDiff('diff'),
-  snapshotDiffDelta('diff-delta'),
-  snapshotDiffNew('diff-new'),
+  snapshotSingle(),
   ;
 
-  const MemoryAreas(this.name);
-
-  final String name;
```

Comment:
@polina-c is it a bug that these are unused?

Reply:
See https://github.com/flutter/devtools/pull/5301/files#r1117500947